### ISSUE TITLE
High priority design refinements

### ISSUE THREAD

**Issue by VanessaVenti** [harvard-lts/CURIOSity#248]
### Masonry View
- [x] New bug: When there is no thumbnail for an item, masonry view now only displays the title when you hover your mouse over the area where it should appear. The title should appear at all times when there is no thumbnail. See [this record](https://curiosity-qa.lib.harvard.edu/slavery-abolition-emancipation-and-freedom/catalog/74-990004753040203941) as an example.
  - This works now, but when there is only one item in the search results with no thumbnail, the item appears over the search results text instead of below the search results line. [See this search as an example](https://curiosity-qa.lib.harvard.edu/slavery-abolition-emancipation-and-freedom/catalog?exhibit_id=slavery-abolition-emancipation-and-freedom&q=990004753040203941&search_field=all_fields). Since this situation is unlikely to occur often, it has been added it to https://github.com/harvard-lts/CURIOSity/issues/249 

### Search box
- [x] In Safari, arrow in search box displays under Search instead of in line.
- [x] In Chrome, when there is fielded searching on an exhibit ([see Demo](https://curiosity-qa.lib.harvard.edu/demo)), EVERYTHING text runs into the carrots.

### Configuration: Tags
- [x] Normalize styles of each tag box (titles should be regular case, buttons and links should be styled consistently)

### Item detail page
- [x] Exhibit tags currently appear as small black buttons; change style so look like links

### Homepage
- [x] Update card link title to read "Explore More Collections" so that it matches link text (under Related Services & Tools section at the bottom of the page)

### Configuration: Order exhibits
- [x] [Order exhibits table](https://curiosity-qa.lib.harvard.edu/exhibits/edit) overlaps with Actions Menu

### Configuration: Items
- [x] “From External Resource” tab (after clicking "Add New"): Rename **Set name** to **Set spec** as this is what actually gets entered here and update the helper text to say 'Type in the set spec to be harvested.' *see https://github.com/scientist-softserv/harvard-curiosity/pull/7

### Page not found
- [x] Update styling of page not found: https://curiosity-qa.lib.harvard.edu/contagionsafs to styling on Library.Harvard: https://library.harvard.edu/collections-exhibits/explore-collectionsaskf. Remove search box and instead suggest users return back to https://curiosity.lib.harvard.edu since it could happen on any exhibit.

### Search: Limit your search filters
- [x] Align start of pagination with search results like HDC (currently pagination is left aligned)

### Curation: Image source
- [x] 'Image source > From this exhibit' should either be hidden (OR grayed out OR update the helper text to say it isn't working currently) from the admin panel because it doesn't work with how we harvest/deliver images currently OR we should harvest/deliver images in a way that this feature can be utilized. See https://github.com/harvard-lts/CURIOSity/issues/31 for more details. 

This option appears under:
   - [x] Appearance > Exhibit Masthead
   - [x] Appearance > Exhibit Thumbnail
   - [x] Browse > Masthead (This whole tab should be hidden regardless, so shouldn't see this one)
   - [x] Browse > Thumbnail
   - [x] Edit mode for a page > Thumbnail tab

Solution for now: Visually hidden from our custom UI. You can still find the "From this exhibit" option in the "Vanilla Spotlight" theme so that troubleshooting can happen easily in the future. Once it's working we can revert the changes we made (see the [changed files](https://github.com/scientist-softserv/harvard-curiosity/pull/11/files)) ([PR with change](https://github.com/scientist-softserv/harvard-curiosity/pull/11))

[Video: Visually hidden “From this exhibit” under Image Source section](https://drive.google.com/file/d/1d9Cg7NwrynuifzxgUgt6YNc1LwsqJ4QN/view?usp=sharing)

## Pull request https://github.com/scientist-softserv/harvard-curiosity/pull/15:
- [x] Hide the "Terms of Use" exhibit card from the collections page, but still make them accessible via URL without logging in (i.e. anyone can get to it via the footer). This is our short-term solution until we get pre-published exhibits working. See https://github.com/harvard-lts/CURIOSity/issues/52 for more details.

## Pull request https://github.com/scientist-softserv/harvard-curiosity/pull/14:
- [x] Hide Item Embed widget from HL theme so admins can still use it in Vanilla for testing a future fix, but curators won't see it and try to use it. (note: item embed widgets already in place will have to be manually removed or updated) See https://github.com/harvard-lts/CURIOSity/issues/230 for more details.

**Comment by VanessaVenti** [comment]
For the masonry view bug, I noticed when there is only one item in the search results with no thumbnail, the item appears over the search results text instead of below the search results line. [See this search as an example](https://curiosity-qa.lib.harvard.edu/slavery-abolition-emancipation-and-freedom/catalog?exhibit_id=slavery-abolition-emancipation-and-freedom&q=990004753040203941&search_field=all_fields). Since this situation is unlikely to occur often, I added it to https://github.com/harvard-lts/CURIOSity/issues/249.
